_episodes/interviews/2023-08-10-luca-palmieri.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Adopting Rust: present and future of the Rust web ecosystem, with Luca Palmieri"
+date: 2023-08-10T10:00:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-08-10-luca-palmieri.mp3
+duration: "47:03"
+length: "48875740"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Marco Otto-Witte discusses with [Luca Palmieri](https://lpalmieri.com) the present and future of the Rust web ecosystem. 
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+- [@00:00] - Introduction
+    - [Video recording of the interview](https://www.youtube.com/watch?v=Xb7NokhAVKI)
+- [@00:27] - Start of the interview
+- [@01:39] - "Zero to Production in Rust"
+    - [Zero to Production in Rust](https://www.zero2prod.com/)
+- [@02:40] - Luca's experience working with Rust at AWS
+- [@19:14] - Scenarios and use cases for Rust adoption
+- [@34:43] - The state of the art and future of web backend development in Rust
+- [@45:57] - Conclusion
+
+### Credits
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Mainmatter](https://mainmatter.com)
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: [Plangora](https://twitter.com/plangora)
```

Comment:
You should set this to yourself, given you wrote these :p

Reply:
I wasn't sure what we meant by "Show Notes" 😂

Reply:
OK, fixed in b8aae8b4d105915854fbced35ffc5cb3e2dc19b4